blog/2024-05-29-oras-1.2.0-and-oras-go-2.5.0/index.mdx

```diff
@@ -0,0 +1,98 @@
+  },
+  "artifactType": "json/example",
+  "referenceByTags": [
+    "$REGISTRY/$REPO:$TAG1",
+    "$REGISTRY/$REPO:$TAG2"
+  ]
+}
+```
+
+Futhermore, if you want to filter out the value of reference and mediaType of the pushed artifact in the standard output, use Go template functions as follows:
+
+```bash
+oras push $REGISTRY/$REPO:$TAG1,$TAG2 sbom.spdx vul-scan.json --format go-template='{{.reference}}, {{.mediaType}}'
+```
+
+```json
+$REGISTRY/$REPO@sha256:4a5b8c83d153f52afdfcb422db56c2349aae3bd5ecf8338a58353b5eb6681c45, application/vnd.oci.image.manifest.v1+json
+```
+
+This feature is still in "Experimental" stage. We welcome feedback and contributions to make this feature more mature. 
+
+### Progress output to show real-time status
+
+Keeping track of manifest content download and upload has never been more intuitive and informative. With the progress output, users can witness the real-time status of manifest content pulling, pushing, and transferring. This is really helpful and effective when pulling or pushing large-size content from or to the registry.
+
+<script src="https://asciinema.org/a/661599.js" id="asciicast-661599" async="true"></script>
+
+### Other enhancements
+
+- Support X.509 mTLS authentication with OCI registries by introducing `--cert-file` and `--key-file` in several ORAS commands
+- Support deletion of manifests and blobs in OCI image layout
+- Introduce `--platform` to oras attach for better multi-arch attaching experience, which allows adding referrer artifact to a specific sub-platform
+- Introduce `oras resolve` to get the digest of an artifact
+
+In addition, the overall user experience and performance are also enhanced in this release：
+
+- Reduce authentication request count for several ORAS commands and support blob mounting across repositories in the same registry for `oras copy`
+- Improve error message based on [ORAS CLI error handling guidance](https://github.com/oras-project/oras/blob/v1.2.0/docs/proposals/error-handling-guideline.md)
+
+There are a few bug fixes and a deprecated feature in this release. For a concrete changelog, please see [ORAS CLI v1.2.0 Release Notes](https://github.com/oras-project/oras/releases/tag/v1.2.0).
+
+### Use ORAS CLI in terminal and CI/CD pipelines
+
+ORAS installation binary is available on Winget, Homebrew, Snap, Docker, and GitHub. It can be installed via one simple command. Please see the installation guide for your environment.
+
+The [ORAS GitHub Actions](https://github.com/marketplace/actions/setup-oras) has been upgraded to ORAS CLI v1.2.0. ORAS CLI has also been integrated with the hosted runner machines ([Ubuntu 22.04](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#cli-tools) and [Ubuntu 20.04](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md#cli-tools)) on GitHub Actions and Azure Devops as a preinstalled software. This delivers out-of-box experience to use ORAS in CI/CD pipelines.
```

Comment:
This is still pending for @wangxiaoxuan273 's PR.

Reply:
Now the setup actions has been upgraded to v1.2.0.